pkg/ansible/run.go

```diff
@@ -64,23 +65,34 @@ func printVersion() {
 func Run(flags *aoflags.AnsibleOperatorFlags) error {
 	printVersion()
 
+	cfg, err := config.GetConfig()
```

Comment:
No need to move this getter up here, move back below.

Reply:
Solved

Reply:
Solved